jsk_fetch_robot/jsk_fetch_startup/euslisp/navigation-utils.l

```diff
@@ -8,6 +8,7 @@
 (ros::load-ros-manifest "jsk_recognition_msgs")
 (ros::load-ros-manifest "power_msgs")
 (ros::load-ros-manifest "sensor_msgs")
+(ros::load-ros-manifest "esp_now_ros")
```

Comment:
Could you write something like the following?
I think that it is probably written to check for the existence of packages that have not been released like switchbot_ros with an if statement.
```suggestion
(if (ros::rospack-find "esp_now_ros")
    (ros::load-ros-manifest "esp_now_ros")
```